dev/com.ibm.ws.microprofile.config.1.1/test/src/com/ibm/ws/microprofile/config/dynamic/test/DynamicSourceCleanupTest.java

```diff
@@ -52,7 +51,7 @@ public void testDynamicSourceCleanup() throws InterruptedException {
 
         // Ensure that the source is being refreshed
         Thread.sleep(refreshInterval + extraInterval);
-        assertThat("Config source was not refreshed", TestDynamicConfigSource.getPropertiesLastCalled, not(equalTo(lastGetPropertiesTime)));
+        assertNotSame("Config source was not refreshed", TestDynamicConfigSource.getPropertiesLastCalled, lastGetPropertiesTime);
```

Comment:
I don't like using `assertNotSame` here for comparing `long`s. The numbers could be large and I'm not sure whether autoboxing guarantees the same instance for the same value.

It's annoying that junit doesn't have an `assertNotEquals` but I think `assertTrue(x != y)` with a message saying what `x` and `y` are would be better. (Or dig into fixing the classpath issue so that you can use the hamcrest matcher).

Reply:
ok